helm/values.yaml

```diff
@@ -81,7 +81,7 @@ queryServiceConfig:
     postgresqlPasswordSecretKey: password
     postgresMaxConnectionAttempts: 200
     postgresConnectionRetryBackoff: 5s
-    trinogresConnectionString: jdbc:trino://localhost:8080/iceberg/iceberg_gcs
+    trinoConnectionString: jdbc:trino://localhost:8080/iceberg/iceberg_gcs
```

Comment:
So, the typo was only in values, right? The key in template is still `trinoConnectionString`

Reply:
yes